src/Apps/Settings/Routes/MyCollection/Components/MyCollectionArtworkGrid.tsx

```diff
@@ -18,6 +19,8 @@ const MyCollectionArtworksGrid: FC<MyCollectionArtworkGridProps> = ({
   artworks,
   onLoadMore,
 }) => {
+  const enableNewSubmissionFlow = useFeatureFlag("onyx_new_submission_flow")
```

Comment:
Note that this feature is already enabled and your changes will be visible when they get deployed. Alternatively, you could use `onyx_post_approval_submission_flow`, which is not yet released.